Fix headings in the Azure section

BrJan opened the pull request:
## Description & motivation
<!---
Describe your changes, and why you're making them. Is this linked to an open
issue, a pull request on dbt core, etc?

To learn more about the writing conventions used in the dbt Labs docs, see the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md).
-->

## To-do before merge
<!---
(Optional -- remove this section if not needed)
Include any notes about things that need to happen before this PR is merged, e.g.:
- [ ] Change the base branch
- [ ] Ensure PR #56 is merged
-->

## Prerelease docs
If this change is related to functionality in a prerelease version of dbt (delete if not applicable):
- [ ] I've added versioning components, as described in ["Versioning Docs"](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/versioningdocs.md)
- [ ] I've added a note to the prerelease version's [Migration Guide](https://github.com/dbt-labs/docs.getdbt.com/tree/current/website/docs/guides/migration/versions)

## Checklist
If you added new pages (delete if not applicable):
- [ ] The page has been added to `website/sidebars.js`
- [ ] The new page has a unique filename

If you removed existing pages (delete if not applicable):
- [ ] The page has been removed from `website/sidebars.js`
- [ ] An entry has been added to `_redirects`
- [ ] [Run link testing](https://github.com/dbt-labs/docs.getdbt.com#running-the-cypress-tests-locally) to update the links that point to the deleted page

BrJan committed:
Update set-up-sso-saml-2.0.md

mirnawong1 commented on website/docs/docs/collaborate/manage-access/set-up-sso-saml-2.0.md:
```diff
@@ -289,7 +289,7 @@ Follow these steps to set up single sign-on (SSO) with dbt Cloud:
 14. Click **Save** at the top of the form.
 
-#### Creating SAML Attributes in AD
+### Creating SAML Settings
```
changing to sentence case per style guide

```suggestion
### Creating SAML settings
```

mirnawong1 reviewed (dismissed):
hey @BrJan thanks for opening this up - one small suggestion

mirnawong1 committed:
Merge branch 'current' into BrJan-patch-3

BrJan committed:
Update website/docs/docs/collaborate/manage-access/set-up-sso-saml-2.0.md

Co-authored-by: mirnawong1 <89008547+mirnawong1@users.noreply.github.com>

BrJan committed:
Merge branch 'current' into BrJan-patch-3

mirnawong1 committed:
Update set-up-sso-saml-2.0.md

@BrJan the table [here](https://docs.getdbt.com/docs/collaborate/manage-access/set-up-sso-saml-2.0#configure-saml-settings-1) was looking wrong so i fixed it too!

mirnawong1 commented:
thanks@ BrJan for this! fyi the table [here](https://docs.getdbt.com/docs/collaborate/manage-access/set-up-sso-saml-2.0#configure-saml-settings-1) was looking wrong so i fixed it too! 
![Screenshot 2022-12-14 at 17 01 52](https://user-images.githubusercontent.com/89008547/207660339-87eeac1a-7a4c-4c9f-9e11-c410685eec1d.png)

it's approved and good for you to merge!

mirnawong1 committed:
Merge branch 'current' into BrJan-patch-3

mirnawong1 committed:
Merge branch 'current' into BrJan-patch-3

mirnawong1 commented:
Hey @BrJan let me know if you want me to merge this for you! 🤗
